### PR TITLE
Document the variant filtering feature

### LINKED ISSUE #3893: Document variant support for Sentry Android Gradle Plugin
### Core or SDK?

Platform/SDK

### Which part? Which one?

sentry android gradle plugin

### Description

Document variant support implemented by https://github.com/getsentry/sentry-android-gradle-plugin/pull/140

### Suggested Solution

Document examples on how to use it in the gradle plugin page https://docs.sentry.io/platforms/android/proguard/#gradle-configuration -> https://github.com/getsentry/sentry-docs/edit/master/src/platforms/android/common/proguard.mdx

### PR DESCRIPTION
**Pull request by cortinico**
Fixes #3893

This PR adds documentation for the variant filtering feature shipped with the Gradle plugin: https://github.com/getsentry/sentry-android-gradle-plugin/pull/140

### CODE REVIEW

File: src/platforms/android/common/proguard.mdx [hunk 1]
```diff
@@ -137,3 +137,33 @@ And, add to `multidex-config.pro` the following lines:
 ```
 
 If you experience issues like `Could not find class` on devices running the `Dalvik VM`, you may expand the above rules to keep the necessary classes in the main dex file.
+
+### Variant Filtering
+
+If you wish, you can specify which variant/flavor/build-type should be ignored by Sentry. Specifically you can do so inside your `app/build.gradle` file:
```
Comment (imatwawana) [inline src/platforms/android/common/proguard.mdx:143]:
```suggestion
You can specify which variant/flavor/build-type should be ignored by Sentry inside your `app/build.gradle` file:
```

**Review by imatwawana — APPROVED** [review]
Made minor wording edit, but otherwise looks good to me (pending technical review). Thank you!

**Review by marandaneto — APPROVED** [review]

**Comment by marandaneto** [comment]
@cortinico thanks so much :)

**Comment by philipphofmann** [comment]
Thanks @cortinico 🥇

**Pull request merged**